pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -382,6 +384,44 @@ where
         Self::prepare_handle(module, 0)
     }
 
+    pub fn gr_reply_deposit(r: u32) -> Result<Exec<T>, &'static str> {
+        let repetitions = r * API_BENCHMARK_BATCH_SIZE;
+        let pid_value_offset = COMMON_OFFSET;
+        let send_res_offset = COMMON_OFFSET + PID_VALUE_SIZE;
+        let mid_offset = send_res_offset + ERR_LEN_SIZE;
+        let res_offset = send_res_offset + ERR_MID_SIZE;
+
+        let module = ModuleDefinition {
+            memory: Some(ImportedMemory::max::<T>()),
+            imported_functions: vec![SysCallName::ReplyDeposit, SysCallName::Send],
```

Comment:
Add comment `gr_send` is required to populate `message_context.outcome.handle` so `gr_reply_deposit` can be called